extensions/wikia/AdEngine/js/video/uiTemplate.js

```diff
@@ -1,47 +1,51 @@
 /*global define*/
 define('ext.wikia.adEngine.video.player.uiTemplate', [
+	'ext.wikia.adEngine.video.player.ui.panel',
 	'wikia.log'
-], function (log) {
+], function (Panel, log) {
 	'use strict';
 
-	var autoPlayLayout = [
+	var createBottomPanel = function (uiElements) {
+			return new Panel('bottom-panel', uiElements);
+		},
+		autoPlayLayout = [
 			'progressBar',
 			'pauseOverlay',
-			'volumeControl',
-			'toggleAnimation'
+			'toggleAnimation',
+			createBottomPanel(['toggleFullscreen', 'volumeControl'])
```

Comment:
I believe we shouldn't change current VUAP behaviour. It's desired for new UAP

Reply:
done